### DISCUSSION TITLE
Ability to set terminal settings globally rather than per machine

### DISCUSSION THREAD

**Discussion by ghost** [isontheline/pro.webssh.net#919] [Ideas]
I like my terminals to use homebrew colour settings (black background, green text) but I have to set it manually on all entries. 

Can we please get global terminal settings?

Merci 👍

**Comment by isontheline** [comment]
You're right. I need to work on it soon : https://github.com/isontheline/pro.webssh.net/issues/488

You will be able to define colors inside the ssh_config file for all your connections.

**Reply by ghost** [reply]
Many thanks.

**Reply by isontheline** [reply]
In a few hours a new beta will be available.
It will let you to set background, foreground and cursor colors from your [ssh_config](https://webssh.net/documentation/help/SSH/ssh-config-file/#special-features)

Example :
```python
Host *
 #!BackgroundColor #ff0000
 #!ForegroundColor blue
 #!CursorColor green
```

I will come back to you when available

**Reply by isontheline** [reply]
@RichardBurt65 Now available inside the latest beta release : 
https://testflight.apple.com/join/QSrBK59z

**Reply by ghost** [reply]
Perfect! Termius sub cancelled and all endpoints moved over to WebSSH.

Only thing left on my "nice to have list" is on left hand column below "Sessions" a history of previous connections :)

Thank you @isonline. I just wish other devs were as responsive as you and your team.

**Reply by isontheline** [reply]
Thank you so much for your feedback and your support @RichardBurt65 🙏
(I'm the only indie dev' behind WebSSH)

> Only thing left on my "nice to have list" is on left hand column below "Sessions" a history of previous connections :)

Would like something like :
* This : https://github.com/isontheline/pro.webssh.net/issues/908
* Or just the ability to show your latest connections launched?

**Reply by ghost** [reply]
Just the ability to have a clickable list of latest connections (as in click to connect).

**Reply by isontheline** [reply]
Thanks you @RichardBurt65 

I have updated this related issue : https://github.com/isontheline/pro.webssh.net/issues/611
